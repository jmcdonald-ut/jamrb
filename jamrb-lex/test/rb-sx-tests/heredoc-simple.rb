<<SIMPLEHEREDOC
This is about as simple as it gets.
SIMPLEHEREDOC
