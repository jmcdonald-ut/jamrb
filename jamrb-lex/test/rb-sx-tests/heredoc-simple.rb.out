((Token ((1 0) on_heredoc_beg "<<SIMPLEHEREDOC"))
 (Token ((1 15) on_nl "\n"))
 (Token ((2 0) on_tstring_content "This is about as simple as it gets.\n"))
 (Token ((3 0) on_heredoc_end "SIMPLEHEREDOC\n")))
